perception/shape_estimation/include/shape_estimation/model/boost_bounding_box.hpp

```diff
@@ -0,0 +1,33 @@
+// Copyright 2018 Autoware Foundation. All rights reserved.
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+#ifndef SHAPE_ESTIMATION__MODEL__BOOST_BOUNDING_BOX_HPP_
+#define SHAPE_ESTIMATION__MODEL__BOOST_BOUNDING_BOX_HPP_
+
+#include "shape_estimation/model/bounding_box.hpp"
+
+class BoostBoundingBoxShapeModel : public BoundingBoxShapeModel
```

Comment:
Thank you for pointing it out. In general, it makes sense to keep related functions in a single class. Since there are only two options for now, I would use a if/else instead.